### PR TITLE
Remove Python 2.6 support

### PR DESCRIPTION
**Pull request by jeremycline**
See https://github.com/fedora-infra/fedmsg/issues/418 and https://github.com/mokshaproject/moksha/pull/48.

Signed-off-by: Jeremy Cline <jeremy@jcline.org>

### CODE REVIEW

File: doc/changelog.rst [hunk 1]
```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+1.0.0
+=====
+
+Backwards incompatible changes
+------------------------------
+
+* Python 2.6 is no longer supported
```
Comment (pypingou) [inline doc/changelog.rst]: Will conflict with #470

**Review by pypingou — APPROVED** [review]
There will be a conflict with #470 but easy enough to fix.

👍 for me

File: doc/changelog.rst (hunk 1 above)
Comment (jeremycline) [reply doc/changelog.rst]: I figured this way I'd be _forced_ to write the release note instead of putting it off until I want to tag the release :smile:

**Comment by jeremycline** [comment]
Not sure what the deal is with codecov here, the report I generate locally shows that line as covered and there's no reason I can see that that code only would have been exercised in Python 2.6 alone.

**Comment by frenzymadness** [comment]
Perfect work! Side note: The specfile should be also modified. Should I create a bug in bugzilla?

**Pull request merged**